lib/foreman_maintain/concerns/hammer_helpers.rb

```diff
@@ -0,0 +1,17 @@
+module ForemanMaintain
+  module Concerns
+    module HammerHelpers
+      def hammer_settings
+        @hammer_settings ||= ForemanMaintain.hammer_settings
+      end
+
+      def self.included(klass)
+        klass.extend(self)
+      end
+
+      def hammer_command
```

Comment:
I would expect this rather being a method taking arguments and performing the commands on it's own, returning the results in some unified format